linode/manager
# upcoming: [DI-16577] - Addition of CloudView Feature

## Description 📝
Added cloudview section to cloud manager
Added namespaces tab to cloudview
Added sample text in paper under namespaces tab
Added feature flag control for cloudview section

## Changes  🔄
List any change relevant to the reviewer.
-Added cloudview section to cloud manager
-Added namespaces tab to cloudview
-Added sample text in paper under namespaces tab
-Added feature flag control for cloudview section

## Preview 📷
**Include a screenshot or screen recording of the change**

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/158276691/68478ae7-2187-4437-abc2-b9d82a9d5442)| ![image](https://github.com/linode/manager/assets/158276691/95543b79-f42b-488e-aac8-4ad5ee4fe01f) |

## How to test 🧪
Navigate to cloud manager and check if cloudview section is added
Check if namespaces tab is displayed on click of cloudview section

### Prerequisites
(How to setup test environment)
- Follow cloud manager standard procedures


### Reproduction steps
(How to reproduce the issue, if applicable)
- N/A

### Verification steps 
(How to verify changes)
Navigate to cloud manager and check if cloudview section is added
Check if namespaces tab is displayed on click of cloudview section

## As an Author I have considered 🤔

*Check all that apply*


---
## Commit message and pull request title format standards

`[upcoming: [DI-161577] - Addition of cloudview section under cloud manager`

---
